content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -312,17 +357,19 @@ metadata:
 ...
 ```
 
-For more information, please see [annotations](/docs/concepts/overview/working-with-objects/annotations/) and [kubectl annotate](/docs/reference/generated/kubectl/kubectl-commands/#annotate) document.
+For more information, please see [annotations](/docs/concepts/overview/working-with-objects/annotations/)
```

Comment:
Can address the use of `please` in another issue.

Reply:
done.